frontend/packages/console-shared/src/selectors/node.ts

```diff
@@ -18,6 +18,9 @@ export const getNodeRoles = (node: NodeKind): string[] => {
   );
 };
 
+export const isLabelPresent = (value: K8sResourceKind, label: string): boolean =>
```

Comment:
`value` makes it sound like the label value.

```suggestion
export const isLabelPresent = (obj: K8sResourceKind, label: string): boolean =>
```

This is too generic for a node selector and should be moved to a more general file like selectors/common.ts

Reply:
`hasLabel` is a better name